uwmweb/uwmcms
# UWM-RAR: Config update for section 554713548

* UWM-RAR: Add header view to make sure no blank row was added.
* UWM-RAR: Remove section 1 that links block causing the header to
  look taller.

## Description:
Remove extra row that made Header look taller.

## Testing:

- Import a current version of Rar's DB from Acquia.
- Run a configuration import to bring in updated node configuration.
- Go to /admin/structure/types/manage/author/display and very the new header view mode is showing up.
- Go to /admin/structure/types/manage/author/display/full/layout and verify section one is not containing a links block.
